dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/task/TaskPluginManager.java

```diff
@@ -53,19 +49,19 @@ public void loadPlugin() {
             logger.warn("The task plugin has already been loaded");
             return;
         }
-        ServiceLoader.load(TaskChannelFactory.class).forEach(factory -> {
-            final String name = factory.getName();
+        PrioritySPIFactory<TaskChannelFactory> prioritySPIFactory = new PrioritySPIFactory<>(TaskChannelFactory.class);
+        for (Map.Entry<String, TaskChannelFactory> entry : prioritySPIFactory.getSPIMap().entrySet()) {
+            String factoryName = entry.getKey();
+            TaskChannelFactory factory = entry.getValue();
 
-            logger.info("Registering task plugin: {}", name);
+            logger.info("Registry task plugin: {} - {}", factoryName, factory.getClass());
```

Comment:
Registry is a noun but here we want a verb, no?

Reply:
We need to use `Registering` rather than `Registry`.